lib/inspec/cli.rb

```diff
@@ -173,6 +173,8 @@ def vendor(path = nil)
     desc: "Generates a tar.gz archive."
   option :overwrite, type: :boolean, default: false,
     desc: "Overwrite existing archive."
+  option :airgap, type: :string, default: false, banner: "", # Actually a boolean, but this suppresses the creation of a --no-airgap...
+    desc: "Prefer using existing local archives to fetching profiles."
```

Comment:
Reading the PR description, it sounds like this option allows a fallback to local files when fetching fails; rather than preferring local files? So maybe `"Allow InSpec to use local archives when fetching dependencies"` ?

The term "local archives" could maybe be more explicitly written as "existing content in the vendor subdirectory"?

Reply:
The behavior is to attempt to fetch first, then fallback (ie, reprogram the fetcher to point at the archive) and re-fetch. If the second fetch fails, give up. 

`Fallback to use local archives if fetching fails`?

Or should there be a `--airgap no-fetch` (or similar) value indicating no first attempt should be made?

Reply:
I was agreeing with the functionality and just critiquing the `desc:` string. It uses the word "prefer" which implies it will fetch local first; which doesn't reflect the functionality.
I'm just suggesting changing this string.

At the same time, it would be awesome to add an option to indicate that InSpec should immediately use the local content first, and not attempt a remote fetch.

e.g.

```
--local-dependencies never
--local-dependencies allow
--local-dependencies always
```

Reply:
OK. Let's leave the functionality alone for now then, so we can see how customers like the feature as-is. Let's revise the usage string to clarify that it is a fallback operation.